### PR TITLE
improve dev doc and port forwarding config

### PR DESCRIPTION
**Pull request by dschlenk**

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -231,16 +296,24 @@ to the main katello server you want the client attached to
...
 :webpack_dev_server_https: true
 ```
 
-This will instruct Foreman to use Webpack from an https source. Now we need to make sure Webpack is using the right certificates to serve the development content through HTTPS. We're going to save these configuration values on `~/foreman/.env`:
+This will instruct Foreman to use Webpack from an https source.
+
+Now recompile webpack:
+
+```
+bundle exec rake webpack:compile
```
Comment (xprazak2) [inline docs/development.md]: I do not think you need to compile again. When using `foreman start`, webpack compiles the assets and servers them on the fly.

File: docs/development.md [hunk 2]
```diff
@@ -52,13 +54,76 @@ Lastly, spin up the box:
...
+Vagrant.configure("2") do |config|
+  config.vm.provider :libvirt do |domain|
+    domain.cpu_mode = "host-passthrough"
+  end
+end
+```
+
+Try to bring it up again with `vagrant up centos7-devel` after making that change.
+
+If you get an error related to your user not having privileges to execute libvirt commands, create a polkit rule similar to the following:
```
Comment (xprazak2) [inline docs/development.md]: 👏

**Comment by xprazak2** [comment]
Looking good, I just left one comment inline.

File: docs/development.md [hunk 3]
```diff
@@ -52,13 +54,76 @@ Lastly, spin up the box:
...
+```
+
+Execute `npm install`:
 
+```
+cd /home/vagrant/foreman
+npm install
+```
+
+Set `:webpack_dev_server` to false in `~/foreman/config/settings.yaml` (until [#136](https://github.com/theforeman/puppet-katello_devel/pull/136)).
```
Comment (ekohl) [inline docs/development.md]: This has been merged.

**Comment by ehelms** [comment]
@ekohl needs your re-review

File: docs/development.md [hunk 4]
```diff
@@ -52,13 +54,71 @@ Lastly, spin up the box:
...
+
+```
+yum install rh-nodejs8-npm http-parser-devel
+```
+
+Enable the nodejs installation:
+
+```
+source /opt/rh/rh-nodejs8/enable
+```
```
Comment (ehelms) [inline docs/development.md]: From my experience, the SCL is not needed to perform npm actions, what's in EPEL is good enough so I'd prefer if we left this out of the change.

**Comment by dschlenk** [comment]
@ehelms @ekohl replaced the SCL stuff with the EPEL equivalent. Haven't tested yet as someone powered off my dev box, but I'll do that as soon as I can get it powered back on. Thanks for your input!

**Comment by ekohl** [comment]
Looks good, but there's a conflict. Apologies for the long review period.

**Comment by ekohl** [comment]
https://github.com/theforeman/forklift/pull/609 will make the webpack aspect redundant.

**Comment by ehelms** [comment]
@dschlenk Apologies for the churn and slowness here. The docs have changed since you last updated. If are able to revisit this we'd still love to get the change in.

**Comment by dschlenk** [comment]
Hi hello @ehelms! No worries, will get back into this shortly.

File: docs/development.md (hunk 2 above)
Comment (ekohl) [reply docs/development.md]: On Fedora 27 it's sufficient to be in the libvirt group since they ship the policy by default.

**Comment by ehelms** [comment]
@dschlenk Any plans to return to this? I'd love to see it in, otherwise will close to clean up open PRs.

**Comment by dschlenk** [comment]
@ehelms I don't have time right now. Go ahead and close it and I'll make a new PR when I find a round tuit.

**Comment by ehelms** [comment]
Thanks for the update @dschlenk !

**Pull request closed without merging**